### DISCUSSION TITLE
Problem with save_visibility_state()

### DISCUSSION THREAD

**Discussion by LPeccoud** [cwapi3d/cwapi3dpython#156] [General]
Hi
how to work with visibility_state : 

vision = visualization_controller.save_visibility_state()
TypeError: Unregistered type : visibility_state

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<pyshell#6>", line 1, in <module>
    vision = visualization_controller.save_visibility_state()
TypeError: Unable to convert function return value to a Python type! The signature was
	() -> visibility_state

same problem with activation_state !

Thanks

**Comment by Brunner246** [comment]
Sorry for the inconvenience. Currently this is not fully implemented for the Python API.
